packages/commonwealth/server/passport/magic.ts

```diff
@@ -143,10 +143,8 @@ async function createNewMagicUser({
   return sequelize.transaction(async (transaction) => {
     const newUser = await models.User.createWithProfile(
       {
-        // never use emails from magic, even for "email" login -- magic maintains the mapping
-        // of emails/socials -> addresses, and we rely ONLY on the address as a canonical piece
-        // of login information.
-        email: null,
+        email: magicUserMetadata.email || null,
```

Comment:
The removed comment seems like a fair enough warning, why did we remove? @mzparacha

Reply:
actually this might be reverted, more in Jake's comment [here](https://github.com/hicommonwealth/commonwealth/pull/7665#issuecomment-2137874205)